### PR TITLE
Added instructions to download the zip file in the Red Hat customer portal

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

**Comment by e-minguez** [comment]
I'm not sure if the build errors are related to this commit, any help?

**Review by matthewhelmke — DISMISSED** [review]
Thank you for adding these instructions.

**Comment by matthewhelmke** [comment]
It looks like the build errors are related to some outdated URLs in other parts of the content and are unrelated to your commit.

File: getting_started/topics/first-boot/distribution-files-product.adoc [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
-
-The 'RH-SSO-{project_version}.[zip|tar.gz]' file is the server-only distribution. It contains only the scripts and binaries
-to run the {project_name} server.
-
-To unpack these files run the `unzip` or `gunzip` and `tar` utilities.
-
+* 'rh-sso-{project_version}.GA.zip'
 
+NOTE: This file can be downloaded in the Red Hat customer portal
+https://access.redhat.com/downloads[Downloads] section, within the "JBoss
```
Comment (stianst) [inline getting_started/topics/first-boot/distribution-files-product.adoc]: Would it not be better to link to https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso

File: getting_started/topics/first-boot/distribution-files-product.adoc [hunk 2]
```diff
@@ -3,14 +3,16 @@
...
+Integration and Automation" subsection.
 
+.Download Page
+image:{project_images}/download-page.png[]
```
Comment (stianst) [inline getting_started/topics/first-boot/distribution-files-product.adoc]: We shouldn't include a screenshot of this. The list of downloads will change from release to release and we can't keep that updated.

**Review by stianst — CHANGES_REQUESTED** [review]
See comments inline

File: getting_started/topics/first-boot/distribution-files-product.adoc (hunk 1 above)
Comment (e-minguez) [reply getting_started/topics/first-boot/distribution-files-product.adoc]: Done

File: getting_started/topics/first-boot/distribution-files-product.adoc (hunk 2 above)
Comment (e-minguez) [reply getting_started/topics/first-boot/distribution-files-product.adoc]: Fixed

**Review by stianst — APPROVED** [review]

**Pull request merged**